apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/TransformerContext.java

```diff
@@ -0,0 +1,23 @@
+package gov.cms.bfd.server.war.commons;
+
+import com.codahale.metrics.MetricRegistry;
+import java.util.Optional;
+
+public class TransformerContext {
+
+  public MetricRegistry metricRegistry;
+  public Object claim;
```

Comment:
I don't really think of the claim as being a part of the context as much as being the thing that is being transformed.